### PR TITLE
MOM 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/blightreaper_thallid_blightsower_thallid.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Oracle:{4}{G/P}: Transform Blightreaper Thallid. Activate only as a sorcery. ({G/P} can be paid with either {G} or 2 life.)
+
+ALTERNATE
+
+Name:Blightsower Thallid
+ManaCost:no cost
+Colors:black,green
+Types:Creature Phyrexian Fungus
+PT:3/3
+T:Mode$ Transformed | ValidCard$ Card.Self | Execute$ TrigDamageAll | Execute$ TrigToken | TriggerDescription$ When this creature transforms into CARDNAME or dies, create a 1/1 green Phyrexian Saproling creature token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/blightreaper_thallid_blightsower_thallid.txt:16]: remove wrong Execute

File: forge-gui/res/cardsfolder/upcoming/hidetsugu_and_kairi.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Name:Hidetsugu and Kairi
+ManaCost:2 U U B
+Types:Legendary Creature Ogre Demon Dragon
+PT:5/4
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw three cards, then put two cards from your hand on top of your library in any order.
+SVar:TrigDraw:DB$ Draw | NumCards$ 3 | StackDescription$ {p:You} draws three cards, | SubAbility$ ChangeZoneDB
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hidetsugu_and_kairi.txt:7]: triggers don't have StackDescription

File: forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+ALTERNATE
+
+Name:Furnace-Blessed Conqueror
+ManaCost:no cost
+Colors:red,white
+Types:Creature Phyrexian Cleric
+PT:3/3
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, create a tapped and attacking token that's a copy of it. 
+SVar:TrigCopy:DB$ CopyPermanent | TokenTapped$ True | TokenAttacking$ True | Defined$ Self | NumCopies$ 1 | AtEOT$ ExileCombat | RememberTokens$ True | SubAbility$ DBCounter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt:17]: Wrong AtEOT

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt:17]: non-existing subability??

File: forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+SVar:DBCounter:DB$ PutCounter | Defined$ Remembered  | CounterNum$ X | CounterType$ P1P1 | SpellDescription$ Put a +1/+1 counter on that token for each +1/+1 counter on CARDNAME. Sacrifice that token at the beginning of the next end step.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt:18]: cleanup missing?

File: forge-gui/res/cardsfolder/upcoming/blightreaper_thallid_blightsower_thallid.txt [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Name:Blightreaper Thallid
+ManaCost:1 B
+Types:Creature Fungus
+PT:2/2
+A:AB$ SetState | Cost$ 3 GP | Defined$ Self | Mode$ Transform | SorcerySpeed$ True | AILogic$ Always | SpellDescription$ Transform CARDNAME. Activate only as a sorcery.
+AlternateMode:DoubleFaced
+Oracle:{4}{G/P}: Transform Blightreaper Thallid. Activate only as a sorcery. ({G/P} can be paid with either {G} or 2 life.)
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/blightreaper_thallid_blightsower_thallid.txt:7]: cost in Oracle is wrong

File: forge-gui/res/cardsfolder/upcoming/hidetsugu_and_kairi.txt [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigDraw:DB$ Draw | NumCards$ 3 | SubAbility$ ChangeZoneDB
+SVar:ChangeZoneDB:DB$ ChangeZone | Origin$ Hand | Destination$ Library | ChangeNum$ 2 | Mandatory$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME dies, exile the top card of your library. Target opponent loses life equal to its mana value. If it's an instant or sorcery card, you may cast it without paying its mana cost.
+SVar:TrigExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ X | ValidTgts$ Opponent | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Defined$ Remembered | ValidSA$ Spell | WithoutManaCost$ True | Optional$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Remembered$CardManaCost
+SVar:Z:Remembered$Valid Instant,Sorcery
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hidetsugu_and_kairi.txt:15]: Z still needs to be checked

File: forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+ALTERNATE
+
+Name:Furnace-Blessed Conqueror
+ManaCost:no cost
+Colors:red,white
+Types:Creature Phyrexian Cleric
+PT:3/3
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, create a tapped and attacking token that's a copy of it. 
+SVar:TrigCopy:DB$ CopyPermanent | TokenTapped$ True | TokenAttacking$ True | Defined$ Self | NumCopies$ 1 | AtEOT$ Sacrifice | RememberTokens$ True | SubAbility$ DBCounter
+SVar:DBCounter:DB$ PutCounter | Defined$ Remembered  | CounterNum$ X | CounterType$ P1P1 | SubAbility$ DBCleanup | SpellDescription$ Put a +1/+1 counter on that token for each +1/+1 counter on CARDNAME. Sacrifice that token at the beginning of the next end step.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sun_blessed_guardian_furnace_blessed_conqueror.txt:18]: Same problem with TriggerDescription done wrong

**Review by tool4ever — APPROVED** [review]

**Review by tool4ever — APPROVED** [review]

**Pull request merged**